pkg/doc/verifiable/presentation_jwt.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import (
+	"fmt"
+
+	"github.com/square/go-jose/v3/jwt"
+)
+
+// JWTPresClaims is JWT Claims extension by Verifiable Presentation (with custom "vp" claim).
+type JWTPresClaims struct {
+	*jwt.Claims
+
+	Presentation *rawPresentation `json:"vp,omitempty"`
+}
+
+func (jpc *JWTPresClaims) refineFromJWTClaims() {
+	raw := jpc.Presentation
+
+	if iss := jpc.Issuer; iss != "" {
```

Comment:
can be simplified `if jpc.Issuer != "" {`

Reply:
Yes, this is smth silly :)